Makefile

```diff
@@ -32,6 +32,9 @@ setup_env:
 setup_env_ug:
 	./scripts/feature_tests/setup_env.sh "ug"
 
+setup_env_no_tls:
+	./scripts/feature_tests/setup_env.sh "no-tls"
```

Comment:
we do not need this file.
we are ok with TLS enabled for now.

Reply:
it can be cleaned up later.

Reply:
It will not make any change to the current features.

Reply:
So we can keep that.